tenable/io/v3/base/endpoints/explore.py

```diff
@@ -34,7 +34,12 @@ def _details(self, obj_id: Union[str, UUID]) -> dict:
 
             >>> tio.{PATHWAY}.details('00000000-0000-0000-0000-000000000000')
         '''
-        return self._get(obj_id, conv_json=self._conv_json)
+        headers: dict = {}
+        if kwargs.get('return_csv', False):
+            headers = {'Accept': 'text/csv'}
+            self._conv_json = True
```

Comment:
@milanaghera-tenable do we need to put conv_json = True in case of 'text/csv' as well?

Reply:
we have used the _conv_json to get the result in string format otherwise we are getting the result in double string format for example `"'filed_1,field_2'"` by using the conv_json parameter we can get the result in string format.